packages/phone-number-privacy/combiner/test/integration/ domain.test.ts

```diff
@@ -0,0 +1,807 @@
+import {
```

Comment:
FYI this file name has leading white space that should be removed

Reply:
should be fixed when https://github.com/celo-org/celo-monorepo/pull/9749 is merged